### PR TITLE
Set up CLI and add tuf repository init

### PR DESCRIPTION
**Pull request by mnm678**
init initializes a TUF repository. In the future, this command will also upload the root and targets metadata to the tuf repository.

### CODE REVIEW

File: tuf-notary/tuf-repository.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package tufnotary
+
+import (
+	"github.com/theupdateframework/go-tuf"
+	util "github.com/theupdateframework/go-tuf/util"
+)
+
+func Init(repository string) error {
+	var p util.PassphraseFunc
+	//TODO: get passphrase
```
Comment (trishankatdatadog) [inline tuf-notary/tuf-repository.go:10]: So key files are not encrypted by default, right?

Comment (mnm678) [reply tuf-notary/tuf-repository.go:10]: Yes, I plan to add encryption here either in an iteration of this pr or a future pr.

**Review by asraa — APPROVED** [review]
looks good to me to start! loop me in on further PRs for sure!

File: tuf-notary/cmd/main.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	"log"
+
+	docopt "github.com/docopt/docopt-go"
+)
+
+func main() {
+	usage := `
+Usage:
+  tuf-notary <command> [<args>....]
+  tuf-notary <command> [--registry=<registry>] [--repo=<repository>]
```
[the remark this answers is not in the document]
Comment (mnm678) [reply tuf-notary/cmd/main.go:14]: Yes, this will be used when the metadata is uploaded to the registry, which is one of the next steps.

**Pull request merged**